cmd/kube-controller-manager/app/certificates.go

```diff
@@ -45,6 +45,7 @@ func startCSRSigningController(ctx ControllerContext) (http.Handler, bool, error
 		return nil, false, nil
 	}
 	if ctx.ComponentConfig.CSRSigningController.ClusterSigningCertFile == "" || ctx.ComponentConfig.CSRSigningController.ClusterSigningKeyFile == "" {
+		klog.V(2).Info("skipping CSR signer controller because no csr cert/key was specified")
```

Comment:
returning false here logs here:
https://github.com/kubernetes/kubernetes/blob/975d0736b35d966c50383f794c0114a34fbcd9e4/cmd/kube-controller-manager/app/controllermanager.go#L525-L528

I guess this adds an extra reason, but we did at least log they were not started, right?